src/sonic-framework/rebootbackend/rebootbe.cpp

```diff
@@ -0,0 +1,284 @@
+#include "rebootbe.h"
+#include <google/protobuf/util/json_util.h>
+#include <unistd.h>
+
+#include <memory>
+#include <mutex>
+#include <string>
+
+#include "logger.h"
+#include "notificationconsumer.h"
+#include "notificationproducer.h"
+#include "reboot_common.h"
+#include "reboot_interfaces.h"
+#include "select.h"
+#include "status_code_util.h"
+
+namespace rebootbackend {
+
+namespace gpu = ::google::protobuf::util;
+
+bool sigterm_requested = false;
```

Comment:
Can we define this in reboot_common?

Reply:
Having it in reboot_common will introduce linker issues in tests folder as it is included in two files.